build.gradle

```diff
@@ -31,14 +31,11 @@ dependencies {
     implementation "com.badlogicgames.gdx:gdx-freetype-platform:$gdxVersion:natives-desktop"
     implementation "com.badlogicgames.gdx:gdx-ai:$aiVersion"
 
-    // JUnit 4, Mockito and Powermock for testing:
-    // https://mvnrepository.com/artifact/junit/junit
-    testImplementation "junit:junit:4.13.2"
+    // JUnit 4
+    testImplementation "junit:junit:4.+"
```

Comment:
Hier war ich wohl zu extrem beim Überfliegen. Das hier bitte nicht

Reply:
Meinst du wirklich den Comment und nicht den anderen mit Autoformatierung in IJ: https://github.com/Programmiermethoden/Dungeon/pull/664#discussion_r1195382047  ? Wenn nicht, warum ist 4.+ gültig, aber 4.13.2 nicht welches aktuell so weit ich erkennen kann identisch ist. Wird hier eine bestimmte Version von JUnit4 benötigt?

Reply:
du hattest doch geschrieben, die versionen sollten stärker beschränkt werden.